.travis/test_connection_to_torodb.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ev
+
+# Wait until torodb it's ready (or timeout after 10s)
+COUNTER=10
+while ! nc -z localhost 27019 && [ $COUNTER -gt 0 ]
```

Comment:
Oh, wait.

I've run a container with the port 27019 open but withou any service running on it and the `nc` success. Probably `toro` isn't running there:

```
$ docker run -d -p 27016:27018 ubuntu sleep 1h
a1ae02a09f8828b02711632edd0958dd1825fc047a5a7a2ee820d3594e3c133c

$ nc -z localhost 27016
Connection to localhost port 27016 [tcp/*] succeeded!
```